build-system/compile/post-closure-babel.js

```diff
@@ -17,6 +17,7 @@
 const argv = require('minimist')(process.argv.slice(2));
 const babel = require('@babel/core');
 const path = require('path');
+/** @type {*} */
```

Comment:
Same comment as above. This type info probably belongs in the `@ampproject/remapping` project. Until it's added there, can we add an exemption?